uninstaller/uninstaller_windows.go

```diff
@@ -0,0 +1,29 @@
+package uninstaller
+
+import (
+	"github.com/pufferpanel/pufferd/logging"
+	"github.com/pufferpanel/pufferd/config"
+	"os"
+)
+
+
+func StartProcess(){
+	deleteFiles()
+}
+
+func deleteFiles(){
+err := os.RemoveAll(config.Get("serverfolder"))
+	if err != nil {
+		logging.Error("Error deleting pufferd server folder, stored in %s",config.Get("serverfolder") , err)

```

Comment:
With logging, if you need String formatting, use the "f" versions (Errorf), although if you are also going to provide an actual error object, it's better to just concat the strings